### DISCUSSION TITLE
Priority link tags

### DISCUSSION THREAD

**Discussion by captain-zsa** [nuxt/nuxt#24626] [Questions]
Hello. I'm use this conf:

```
css: [
    '~/assets/main.css',
  ],
  app: {
    head: {
      link: [
        {
          rel: 'stylesheet',
          href: `${process.env.API}/unique/${process.env.SITE}/styles.css`,
        },
      ],
    },
  },
```

In my main.css I have :root { --color-main: #000; }

In my link  styles.css I have :root  { --color-main: #fff; }

Now nuxt add link tags in the order:

```
<link href=".......styles.css"/>
...any tags here...
<link href="......main.css" />
```

And `:root {}` in my `main.css` is more specificly, but I want the opposite. How do it?

**Comment by xjccc** [comment]
use `tagPriority` to control the build position?

**Reply by captain-zsa** [reply]
How write tagPriority for css:[] ?)) In head: link I can do it. And can write tagPriority: 'after:{key_into}'. But maybe can I create key for css:[]?

**Reply by xjccc** [reply]
like this?
```
link: [
        {
          rel: 'stylesheet',
          href: 'style.css',
          tagPriority: 'low'
        }
 ]
```

**Reply by captain-zsa** [reply]
I use '~/assets/main.css', in css:[] for added general styles (scss mixins, variables and more). If I setup scss in link[], can I use scss vars/mixins in my app anywhere (.vue components)?

**Reply by xjccc** [reply]
i try to set tagPriority low. i find style.css below the main.css.(main.css in css[]) it the position you want. you can try it.
